### ISSUE TITLE
Label height changes with text

### ISSUE THREAD

**Issue by IceflowRE** [godotengine/godot#70279]
### Godot version

v4.0.beta8.official [45cac42c0]

### System information

Windows 10

### Issue description

When adding text to a label the height changes, even with simple text like a space.
I would expect that the label height does not change, like it behaves in Godot 3.

### Steps to reproduce

Create a label.
Add text.

### Minimal reproduction project

N/A

**Comment by IceflowRE** [comment]
Applies to buttons as well.

**Comment by Calinou** [comment]
I remember this being intentional, so that icon-only buttons can be made shorter. However, your button needs to be made tall enough for it to be able to accomodate text.

**Comment by IceflowRE** [comment]
> I remember this being intentional, so that icon-only buttons can be made shorter. However, your button needs to be made tall enough for it to be able to accomodate text.

Yeah ok that sounds reasonable, but regarding labels i think it is sill an issue because, it happens more often that you have a label without text. 
For example having always the same spacing in a table, you need an empty label as a placeholder, so the layout does not change.

**Comment by Calinou** [comment]
> For example having always the same spacing in a table, you need an empty label as a placeholder, so the layout does not change.

For "spacer" nodes in containers, use a Control with a minimum size greater than 0.

**Comment by IceflowRE** [comment]
> 

I feel an easier way would be to just add a space or other invisible text to the label. Then i do not have to worry if the font size changes.

**Comment by IceflowRE** [comment]
The line spacing is ignored if a label has no text.
Using a line spacing of 0 will remove the jumpiness of the label size.

**Comment by timothyqiu** [comment]
I think this is a bug. Line spacing show be ignored for the last line.
